include/zck.h.in

```diff
@@ -185,6 +185,9 @@ char ZCK_PUBLIC_API *zck_get_range_char(zckCtx *zck, zckRange *range)
 /* Get file descriptor attached to zchunk context */
 int ZCK_PUBLIC_API zck_get_fd(zckCtx *zck)
     ZCK_WARN_UNUSED;
+/* Get context from zchunk */
```

Comment:
For clarity in the function name and a clear comment, let's do the following:
```
/* Get context from chunk */
zckCtx ZCK_PUBLIC_API *zck_get_chunk_ctx(zckChunk *chunk)
    ZCK_WARN_UNUSED;
```

Reply:
Sounds excellent, thank you for the quick reply.
Please let me know if you want me to create a new pull request with the suggested change.

Reply:
No need to do a new PR, just update this one, ideally amending this commit.